src/page-unsupported-browser/UnsupportedBrowserPage.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+import SiteHeader from "../site-header/SiteHeader";
+
+const GetInvolvedPageDiv = styled.div``;
+
+const UnsupportedBrowserPage: React.FC = () => (
```

Comment:
Fantastic stuff, @szhu! Here is the finalized copy for the page, then we're good to go:




We're sorry, legacy browsers such as Internet Explorer are not currently supported by the COVID-19 model. Please try loading the website in a modern browser such as Microsoft Edge, Google Chrome, Firefox, or Safari.

If you think you are seeing this message in error, or if you can't switch browsers, please email us at <a href="mailto:covid@recidiviz.org?Subject=COVID%20Model%20Browser%20Support" className="font-semibold">covid@recidiviz.org</a> so we can provide support.

Reply:
ah, forgot about Safari!

![image](https://user-images.githubusercontent.com/1570168/79012753-e5ebf100-7b1b-11ea-9396-07386a77e705.png)